src/models/m_config.erl

```diff
@@ -191,7 +191,9 @@ set_value(Module, Key, Value0, Context) ->
                 "Configuration key '~s.~s' changed, new value: '~s', old value '~s'",
                 [ z_convert:to_binary(Module), z_convert:to_binary(Key), Value, OldV ],
                 [ {module, ?MODULE}, {line, ?LINE} ],
-                Context)
+                Context);
+        {rollback, _} ->
+            ok
     end,
```

Comment:
If we have a rollback - shouldn't we then retry?

Reply:
Made it a bit more restrictive and added ok/error return values to the function.